openapi/publicapi_spec.json

```diff
@@ -1064,6 +1375,65 @@
           }
         }
       },
+      "Agreement" : {
+        "type" : "object",
+        "properties" : {
```

Comment:
Missing descriptions and examples for the properties in the response. Cross reference with the equivalent payment endpoints for the level of detail we should include.